docs/book/getting_started/hello_world.md

```diff
@@ -1,92 +1,117 @@
-{% panel style="info", title="Under Development" %}
-This book is being actively developed.
-{% endpanel %}
-
 # Hello World
```

Comment:
Instead of Hello World or in addition to?

Reply:
applicationcontroller?

Reply:
> Instead of Hello World or in addition to?
Instead of Hello World. Main concern was -- do we have a good bridge from `patterns` --> `mgr/controller/Source` if a user plan to use lower level stuff ?.

Reply:
Renamed